src/pages/MemberProfileScreen.tsx

```diff
@@ -0,0 +1,23 @@
+import { Container, Divider, Stack, Typography } from '@mui/material';
+
+import MemberPersonalInformation from '@/components/main/MemberPersonalInformation';
+import MemberPublicProfile from '@/components/main/MemberPublicProfile';
+import { useAccountTranslation } from '@/config/i18n';
+
+const MemberProfileScreen = (): JSX.Element => {
+  const { t } = useAccountTranslation();
+  return (
+    <Container sx={{ maxWidth: '1000px' }}>
```

Comment:
No need, Container has by default a maxWidth of 1200.
```suggestion
    <Container>
```